aws/eb_rds/main.tf

```diff
@@ -0,0 +1,86 @@
+# Create a security group for the beanstalk server
+resource "aws_security_group" "beanstalk" {
+  name   = "${var.eb_application}-${var.eb_environment}-server-sg"
+  vpc_id = "${module.vpc.vpc_id}"
+
+  ingress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["${module.vpc.public_subnets_cidr}"]
```

Comment:
Why only allowing traffic from the public subnet? If the environment is single instance won't we need access from anywhere?